src/components/Sum.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => {
+  const text = `Sum of ${a} and ${b} is ${a + b}`;
+
+  return (
+    <p>{text}</p>
```

Comment:
Why do you need a variable here?
```suggestion
    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```